my-app/src/app/blog/carousel/carousel.component.css

```diff
@@ -0,0 +1,20 @@
+div {
+    display: flex;
+    flex-direction: row;
+    background-color: black;
+    height: auto;
+}
+
+button {
+    margin-top: 8rem;
+    color: white;
+
+}
+iframe {
+    padding-top: 9rem;
```

Comment:
Certain images/facebook posts are shorter and have some of the black background showing under, others are longer and don't. Can you be consistent and make all of them either have the black showing under each post, or not.